components/Sidebar.tsx

```diff
@@ -34,6 +35,7 @@ export const Sidebar = () => {
       <LinkButton href="https://github.com/lucavallin/first-issue#adding-a-new-project">
         Add your project
       </LinkButton>
+      <CollectivSection />
```

Comment:
I am not a fan of this one, it really pushes the browse tabs way down ;) alternative locations?

Reply:
Hey @lucavallin, we tried a few design iterations but other places were not looking good as well. Suggestion - we can keep this for October and till then really try a few more iterations on design for an alternative place.

If this this cannot go for October as well, then we can go ahead and comment it out.

Reply:
Sounds good!